src/lib/pipeline.ml

```diff
@@ -392,6 +402,8 @@ let rec to_file_prefix:
       sprintf "%s-%s-%s" prev
         vc.Variant_caller.name
         vc.Variant_caller.configuration_name
+    | Rna_hla_typer s ->
+      sprintf "RNA_HLA_typer-%s" (to_file_prefix ?read s)
```

Comment:
Please see message above about using `Rna_hla_typer` as the variant name. At the moment, there is only _one_ typer, so the message should suffice.